test-data/unit/python2eval.test

```diff
@@ -5,7 +5,7 @@
 -- complete.
 
 
-[case testAbs2_python2]
+[case testAbs2_python2eval]
```

Comment:
I really would rather not have to rename all the tests to conform to this new naming convention. I think you can select for just the pythoneval tests with `-k PythonEvaluationSuite`, so to combine these two you can run
```
pytest -k "PythonEvaluationSuite and python2"
```
(the Windows script will require some extra quoting).

Reply:
Okay I will try that route.